backend/test/gpml/handler/browse_test.clj

```diff
@@ -214,4 +214,11 @@
         ;; only tech returned
         (is (= 1 (count counts)))
         (is (= (:topic tech-count) "technology"))
-        (is (= (:count tech-count) 1))))))
+        (is (= (:count tech-count) 1))))
+
+    (testing "Testing query for BOGUS tags"
+      (let [resp (handler (-> (mock/request :get "/")
+                              (assoc :parameters {:query {:tag "bogus1,bogus2"}})))
+            body (-> resp :body)
+            results (:results body)]
+        (is (= 0 (count results)))))))
```

Comment:
I think it could be good to test on positive case too

Reply:
Agreed!